src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProvidersController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.RoatpCourseManagement.Application.Standards.Queries.GetProvider;
+
+namespace SFA.DAS.RoatpCourseManagement.Api.Controllers
+{
+    [ApiController]
+    public class ProvidersController : ControllerBase
+    {
+        private readonly ILogger<ProvidersController> _logger;
+        private readonly IMediator _mediator;
+
+        public ProvidersController(ILogger<ProvidersController> logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [HttpGet]
+        [Route("providers/{ukprn}")]
+        public async Task<IActionResult> GetProvider([FromRoute] int ukprn)
+        {
+            if (ukprn <= 9999999)
```

Comment:
I know we have done this elsewhere... but we can avoid any validation in the outer api as this is being validated on both web and inner.

Reply:
I'll take it out